auditor/compiler.py

```diff
@@ -0,0 +1,127 @@
+import os
+import importlib
+from auditor.base_exceptions import CompileException
+from itertools import chain
+
+class AuditorCompiler(object):
+    version = '1'
+    operator_tokens = [
+        'read',
+        'remove_bad_data',
+        'encoding',
+        'separator',
+        'quotechar',
+        'write',
+        'column_add',
+        'column_order',
+        'column_rename',
+        'col',
+        '|',
+    ]
+
+    def __init__(self):
+        pass
+
+    def __call__(self, program_file_path):
+        self.program_file_path = program_file_path
+        with open(program_file_path, 'r') as program_file:
+            self.code = program_file.read()
```

Comment:
Probably don't need self.code here since it's only used in the next line. I'm sure it doesn't add any overhead, just something I noticed